PyRDF/RDataFrame.py

```diff
@@ -183,6 +183,29 @@ def get_inputfiles(self):
         # RDataFrame may have been created with no input files
         return None
 
+    def get_input_friend_treenames_and_filenames(self):
```

Comment:
Even though I can understand why you put it here, I would not implement this method in `RDataFrame.py` but in `Dist.py`. 

While the rest of `get` methods (`get_inputfiles`, `get_treename`, `get_branches` ... ) need to check the parameters passed to the `RDataFrame` constructor in order to access to the required information (input files, treename, branches,...), this method could be directly applied to a `TTree/TChain` as it is done in the ROOT implementation. Indeed, this method will be only called from a `Dist` class or subclass.